src/Libraries/CoreNodes/List.cs

```diff
@@ -1257,7 +1257,20 @@ public static object LastItem(IList list)
         [IsVisibleInDynamoLibrary(true)]
         public static IList Shuffle(IList list)
         {
-            var rng = new Random();
+            return list.Cast<object>().OrderBy(_ => mRandom.Next()).ToList();
```

Comment:
As commented earlier, is this not a breaking change in terms of breaking existing graph behaviour?

Reply:
Existing behavior is non-breaking in any singular situation (i.e. using the List.Shuffle node once). 

In any dual behavior situations, the current node is **not working as expected** by the vast majority of users and shuffling uniquely. It is also incredibly difficult to find out why (For the average user) as you need to look into the codebase itself and understand that the .NET Framework default seed value is derived from the system clock - meaning any concurrently executing list.shuffle nodes with the same input data will shuffle to the same results. 

While technically this can be a breaking change, it's currently more expensive for users to try and resolve than necessary. You have to resort to Python, a custom node or create your own C# node which is beyond many users, either through creation or simply finding it. 

With the addition of the nodes above from @saintentropy, the 'fix' to retain existing behavior is simply to add the same seed value.